README.md

```diff
@@ -1,8 +1,20 @@
 # Interplay - Play Build plugins
 
-Interplay is a set of sbt plugins for Play builds, sharing common configuration between Play builds so that they can be configured in one place.
+[![Twitter Follow](https://img.shields.io/twitter/follow/playframework?label=follow&style=flat&logo=twitter&color=brightgreen)](https://twitter.com/playframework)
+[![Discord](https://img.shields.io/discord/931647755942776882?logo=discord&logoColor=white)](https://discord.gg/g5s2vtZ4Fa)
+[![GitHub Discussions](https://img.shields.io/github/discussions/playframework/playframework?&logo=github&color=brightgreen)](https://github.com/playframework/playframework/discussions)
+[![StackOverflow](https://img.shields.io/static/v1?label=stackoverflow&logo=stackoverflow&logoColor=fe7a16&color=brightgreen&message=playframework)](https://stackoverflow.com/tags/playframework)
+[![YouTube](https://img.shields.io/youtube/channel/views/UCRp6QDm5SDjbIuisUpxV9cg?label=watch&logo=youtube&style=flat&color=brightgreen&logoColor=ff0000)](https://www.youtube.com/channel/UCRp6QDm5SDjbIuisUpxV9cg)
+[![Twitch Status](https://img.shields.io/twitch/status/playframework?logo=twitch&logoColor=white&color=brightgreen&label=live%20stream)](https://www.twitch.tv/playframework)
+[![OpenCollective](https://img.shields.io/opencollective/all/playframework?label=financial%20contributors&logo=open-collective)](https://opencollective.com/playframework)
+
+[![Build Status](https://github.com/playframework/interplay/actions/workflows/build-test.yml/badge.svg)](https://github.com/playframework/interplay/actions/workflows/build-test.yml)
+[![Maven](https://img.shields.io/maven-central/v/com.typesafe.play/interplay_2.13.svg?logo=apache-maven)](https://mvnrepository.com/artifact/com.typesafe.play/interplay_2.13)
```

Comment:
mvnrepository.com return 404 for the interplay artifact, but I think that is a bug from their side. The package is here: https://repo1.maven.org/maven2/com/typesafe/play/interplay_2.12_1.0/ but I think mvnrepository.com can not handle the `_1.0` suffix.